app/models/restaurant.rb

```diff
@@ -0,0 +1,15 @@
+class Restaurant < ActiveRecord::Base
+
+  # Cuisine type
+  CUISINE = {meat: 0, asian: 1}
+
```

Comment:
We usually do not convert the enums to int in the DB here... not that there is anything wrong with it..